tweets.py

```diff
@@ -1,25 +1,33 @@
 import requests
 import csv
 from pandas import Timestamp, Timedelta
-from utils import list_average
+from utils import list_average, get_reader
 
 DATA_URL = 'https://www.istheweatherweird.com/istheweatherweird-data-hourly'
 STATIONS_URL = '{}/csv/stations.csv'.format(DATA_URL)
 
-CITY = 'Chicago'
 START_TIME = Timestamp.utcnow().replace(microsecond=0) - Timedelta(days=1)
 END_TIME = Timestamp.utcnow().replace(microsecond=0)
 
+def get_tweets(city):
+    place = get_place(city)
+    local_time = END_TIME.tz_convert(place['TZ'])
 
-def get_tweets():
-    place = get_place(CITY)
+    tweets = []
 
-    daily_temp = get_daily_temp(place)
-    historical_temps = get_historical_temps(place)
+    # check if 6pm <= current local time < 7pm
+    # this is intended to run every hour through the Heroku scheduler
+    if local_time.floor(freq='h').hour == 18:
```

Comment:
`pandas.Timestamp` objects have an `hour` attribute so you can just do `if local_time.hour == 18:`